docs/synapse/devguides/storm_api.rst

```diff
@@ -435,6 +440,20 @@ Example:
                  'c5a67a095b71771d9663d691f0ab36b53ebdc14fbad18f23f95e923543156bd6',)
         opts = {'idens': idens}
 
+keepalive
+---------
+
+This is the period ( in seconds ) in which to send a ``ping`` message from a Storm query which is streamiing results,
+such as the Telepath ``.storm()`` API or the HTTP ``/v1/api/storm`` API endpoint. The value must be greater than zero.
```

Comment:
I'd add a single line about the use case for long-running queries and proxies or load balancers.